concrete/src/Authentication/Type/OAuth/GenericOauthTypeController.php

```diff
@@ -512,4 +519,28 @@ public function getUniqueId()
     abstract public function handle_attach_attempt();
 
     abstract public function handle_attach_callback();
+
+    public function handle_detach_attempt()
+    {
+        $user = new User();
+        if (!$user->isLoggedIn()) {
```

Comment:
This method is static, you can just do `\Concrete\Core\User\User::isLoggedIn()`

Reply:
Fixed on https://github.com/concrete5/concrete5/pull/6018/commits/f162516be17421c2a0a5804868c63f839ec27f80